config/default.yml

```diff
@@ -2,7 +2,7 @@ allowUnused: []
 
 devFilePatterns:
   - '{features,spec,test}/**/*'
-  - '**/*_{spec,test}.js'
+  - '**/*{spec,test}.js'
```

Comment:
I believe it would need to be `'**/*{.,_,-}{spec,test}.js'`

We already use `'**/*{spec,test}.js'` on mycha. We should update that to match.